### ISSUE TITLE
What do we do with learn more links for MVP?

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#979]

**Comment by colinmurphy01** [comment]
@kmranjo heads up on this

**Comment by kmranjo** [comment]
We actually have a third one too: for any WFO promo component that's been authored. For that one, I think I put in a ticket that allows us to author the link manually or remove it, instead of having a link that doesn't work. 

For these two, we will likely either need to link to existing pages or remove the links. 
For obs, there is this page, which doesn't hit the intended target exactly, but is close: https://www.weather.gov/about/observations 

For precip, similarly, there's a page that a WFO created that we can link to, but in the long term we'd want to repackage content like this to be more general https://www.weather.gov/lox/rainrate

**Comment by kmranjo** [comment]
Also to consider: Ideally we don't want the old and new site to have too much crossover. The more separate they are, the more the new site is solidly established as a new experience for users, and the fewer points of crossover we would have to go back and clean up later when the newer version of the page is created. But if there's content that is valuable enough to benefit the user experience even when being on the old site, we can include those links. 

For the 3 mentioned above, I think the only one that might provide that value is the precip guide. The WFO pages aren't that useful in current state, and the obs stations page only really talks about the COOP stations.

**Comment by colinmurphy01** [comment]
If we were to only link to the precipitation page, what should we do with the obs link? Remove it for MVP?

**Comment by kmranjo** [comment]
Yeah, I think we'd have to take it out and just plan to put it back once the page exists

**Comment by colinmurphy01** [comment]
For the time being, we have decided to not link back to weather.gov and will remove these placeholder links.

There isn't enough justification right now to link back to weather.gov materials to warrant the confusing user experience of switching between sites.
